lighthouse-core/test/gather/gather-runner-test.js

```diff
@@ -423,6 +425,41 @@ describe('GatherRunner', function() {
     });
   });
 
+  it('warns if important data not cleared may impact performance', () => {
+    const asyncFunc = () => Promise.resolve();
+    driver.assertNoSameOriginServiceWorkerClients = asyncFunc;
+    driver.beginEmulation = asyncFunc;
+    driver.enableRuntimeEvents = asyncFunc;
+    driver.enableAsyncStacks = asyncFunc;
+    driver.cacheNatives = asyncFunc;
+    driver.registerPerformanceObserver = asyncFunc;
+    driver.dismissJavaScriptDialogs = asyncFunc;
+    driver.registerRequestIdleCallbackWrap = asyncFunc;
+    driver.clearDataForOrigin = asyncFunc;
+
+    connectionStub.sendCommand = createMockSendCommandFn()
+      .mockResponse('Storage.getUsageAndQuota', {usageBreakdown: [
+        {storageType: 'local_storage', usage: 5},
+        {storageType: 'indexeddb', usage: 5},
+        {storageType: 'websql', usage: 0},
+        {storageType: 'appcache', usage: 5},
+        {storageType: 'cookies', usage: 5},
+        {storageType: 'file_systems', usage: 5},
+        {storageType: 'shader_cache', usage: 5},
+        {storageType: 'service_workers', usage: 5},
+        {storageType: 'cache_storage', usage: 0},
+      ]});
+    /** @type {string[]} */
+    const LighthouseRunWarnings = [];
+    GatherRunner.setupDriver(driver, {settings: {}}, LighthouseRunWarnings).then(_ => {
+      expect(LighthouseRunWarnings[0]).toBeDisplayString(new RegExp(
```

Comment:
I'm getting this error in this test file for some reason. The other uses of `expect(...).toBeDisplayString()` in this file use regex notation instead of string literal.
<img width="725" alt="Screen Shot 2020-09-24 at 12 48 46 PM" src="https://user-images.githubusercontent.com/6752989/94175170-4c639200-fe64-11ea-9fcc-ee522a005398.png">

Reply:
Ah, looks like we got the type declaration wrong and we just don't type check all the other test files that use this :)

https://github.com/GoogleChrome/lighthouse/blob/bc6ab76a33f50403e4bfc84eb069b070ed10466f/types/jest.d.ts#L23

should include `string | `

Reply:
Is this a fix I should include in this PR?

Reply:
> Is this a fix I should include in this PR?

It's such a small extra change, it's only for the type on code that can only ever be used in tests, and it's additive rather than changing the type, so it seems fine to me to land in this PR, but feel free to split it out if you'd prefer.